tests/Tutor.Infrastructure.Tests/TestData/EventStore/TestEventSerializer.cs

```diff
@@ -0,0 +1,45 @@
+﻿using Dahomey.Json;
+using System;
+using System.Collections.Generic;
+using System.Text.Json;
+using Tutor.Core.BuildingBlocks.EventSourcing;
+using Tutor.Infrastructure.Database.EventStore;
+using Tutor.Infrastructure.Serialization;
+
+namespace Tutor.Infrastructure.Tests.TestData.EventStore
+{
+    public class TestEventSerializer : IEventSerializer
```

Comment:
Implementing this for the tests made me realize that the way the event serialization is implemented should probably be refactored a bit. It might be good to move the implementation with Dahomey.Json to the EventStore instead to offer a DefaultEventSerializer. 
I'm not sure if I should do that now since it's a relatively small change, or some other time?

Reply:
Implemented in 2d432ad1cd6fdc9bad2beab8cffba35c44fa76dd.